docs/programming-oak.md

```diff
@@ -407,13 +407,17 @@ associated type to
 or manually reading from the initial `Receiver`:
```

Comment:
Could you update this doc? I think we don't need to mention the initial Receiver any more, all nodes are expected to impl CommandHandler now.

Reply:
Deleted `or manually reading from the initial Receiver`